.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We probably want to populate this and then point all our VS Code Prettier settings to use this from the project.

But we want to check that there's not a base prettier.rc that should take precedence for the entire workspace first.